clients/ts/FunctionalTests/Startup.cs

```diff
@@ -103,6 +103,16 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 routes.MapConnectionHandler<EchoConnectionHandler>("/echo");
             });
 
+            app.Use(async (context, next) =>
+            {
+                if (context.Request.Path.Value.Contains("/negotiate"))
+                {
+                    context.Response.Cookies.Append("testCookie", "testValue");
```

Comment:
Can you put two valid cookies in? I'm paranoid about `getCookieString` only returning one cookie or something :)